.github/workflows/backup-artifact-linux.yml

```diff
@@ -1,7 +1,7 @@
 name: Build:Main
 on:
   push:
-    branches: [ master, test-ready ]
+    branches: [ master ]
```

Comment:
you sure you don't want the extra check?

Reply:
This workflow sets the latest tag. I don't want it set to any commits on test-ready.